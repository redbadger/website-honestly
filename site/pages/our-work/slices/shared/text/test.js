// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Text from '.';

describe('site/pages/our-work/slices/shared/text', () => {
  it('renders the correct layout class by default', () => {
    const component = shallow(<Text>hey joe</Text>);
    expect(component).toMatchSnapshot();
  });
  it('renders the correct layout class', () => {
    const component = shallow(<Text layout="fullWidth">hey joe</Text>);
    expect(component).toMatchSnapshot();
  });
});
