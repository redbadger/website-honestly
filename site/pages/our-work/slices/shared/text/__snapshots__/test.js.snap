// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/slices/shared/text renders the correct layout class 1`] = `
<div
  className="fullWidth"
>
  hey joe
</div>
`;

exports[`site/pages/our-work/slices/shared/text renders the correct layout class by default 1`] = `
<div
  className="leftCol"
>
  hey joe
</div>
`;
